link2aws.js

```diff
@@ -12,6 +12,7 @@ class ARN {
         }
 
         // arn:partition:service:region:account-id:...
+        this.arn = text;
```

Comment:
Can you update the tests in `test/test.js` that check all the fields to check the new `.arn` field as well?